content/english/blog/new_logo_2025.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "PyPodcats Have a New Logo"
+meta_title: "PyPodcats Have a New Logo"
+description: "Finally after 1 year, we finally have a logo"
+date: 2025-06-17T05:00:00Z
+categories: ["Blog Post",]
+author: "Georgi"
+tags: ["python", "community", "women in tech", "women in Python", "women in open source","python underrespresnted groups"]
+draft: false
+image: "/images/logo-blog.png"
+---
+
+
+After a year of placeholder art and indecision, we finally have a real logo. And yes—it has a cat. On a mic. With a snake. We promise it makes sense.
+Our original stand-in was a charming sticker from Mariatta’s [“#TypoOfTheDay”](https://mariatta.ca/posts/typo_of_the_day/) series. It wasn’t meant to be a logo, but we were so eager to get *Hidden Figures of Python* out into the world that we just rolled with it. MVP: Minimum Viable Purr.<br>
+Many drafts, discussions, and digital furballs later, our team chose the new design: a laid-back cat draped over a retro microphone, with a happy snake coiled in—our nod to the Python community that brought us all together.
+We printed our first batch of stickers just in time for PyCon US this past May. They were glittery. They were gorgeous. They were gone in a flash. Sorry if you missed them—we had a tight budget and only printed a small run using our own pocket money. Blame the import tax on sparkles.
+{{<figure src="/images/new-sticker-2025.jpg" caption="*New glittery PyPodcats sticker printed for PyConUS 2025*" alt="PyPodcats sticker for PyconUS 2025">}}
```

Comment:
Something is weird in the preview, eg the next paragraph has a different text color compared to the rest.
I wonder if it is because of the `*` in the caption?
Maybe try removing the * and see.
```suggestion
{{<figure src="/images/new-sticker-2025.jpg" caption="New glittery PyPodcats sticker printed for PyConUS 2025" alt="PyPodcats sticker for PyconUS 2025">}}
```
![Screenshot 2025-06-23 at 4 14 51 PM](https://github.com/user-attachments/assets/7c1d1543-b5f4-4874-93ad-c36dacfd53ab)